### PR TITLE
React week1

### PR DESCRIPTION
**Pull request by hparul**

### CODE REVIEW

File: react/week1/my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+//import logo from './logo.svg';
+import './App.css';
+import './ToDoItem.css';
```
Comment (islanderupnorth) [inline react/week1/my-app/src/App.js:3]: Minor adjustment but .css files are usually done kebab-case style

File: react/week1/my-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import ToDoList from './ToDoList.js';
+
+
+function App() {
+
+  return (
+
+    <div className="App">
+      <header className="App-header">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTIJoynbf7Cqar_m8fr8VWdKIVnDT-oampspA&usqp=CAU" alt=" girl with list" className="App-logo" />
```
Comment (islanderupnorth) [inline react/week1/my-app/src/App.js:17]: I couldn't run the code but couldn't help but take a look at what this image was - nice touch!

**Review by islanderupnorth — COMMENTED** [review]
Overall great structure how you gave the list and items separate files.

As a part of this homework if you could get it running on your local environment using create-react-app (https://github.com/facebook/create-react-app) either before or after class that would be great

Javascript & React wise it looks good!

File: react/week1/my-app/src/App.js (hunk 1 above)
Comment (hparul) [reply react/week1/my-app/src/App.js]: done with that

File: react/week1/my-app/src/App.js (hunk 2 above)
Comment (hparul) [reply react/week1/my-app/src/App.js:17]: Hi, I could run the code on local host using create-react-app already.can you please go through the code again to run it.
thanks for your feedback!

**Pull request merged**